### DISCUSSION TITLE
Canary's macOS version updates slower than the Windows version

### DISCUSSION THREAD

**Discussion by pipashu** [SpyNetGirl/MSEdgeFeatures#9] [General]
Can you update Canary's macOS and Windows versions simultaneously?

**Comment by HotCakeX** [comment]
Hi,
I'm not sure since I don't use Mac but I assume yes, to be sure you can compare the latest version that's shown in this GitHub repository with your Edge canary version on Mac. This repository always shows the latest version of Edge canary on Windows.
